dbt_project.yml

```diff
@@ -1,7 +1,7 @@
 name: 'dbt_artifacts'
 version: '1.2.0'
 config-version: 2
-require-dbt-version: ">=1.0.0"
+require-dbt-version: ">=1.2.0"
```

Comment:
What's the reason for this?

Reply:
The cross database type macros are a post 1.2 feature

Reply:
I believe it's because the [cross-database macros](https://docs.getdbt.com/reference/dbt-jinja-functions/cross-database-macros) being used in the new sources (E.g. [type_string()](https://docs.getdbt.com/reference/dbt-jinja-functions/cross-database-macros#type_string), [type_timestamp()](https://docs.getdbt.com/reference/dbt-jinja-functions/cross-database-macros#type_timestamp)) are only available in 1.2+

Reply:
damn @jaypeedevlin beat me